functional-test/src/main/java/org/zanata/page/WebDriverFactory.java

```diff
@@ -107,11 +108,17 @@ private WebDriver createPlainDriver() {
     }
 
     private WebDriver configureChromeDriver() {
+        // TODO can we use this? it will use less code, but it will use DISPLAY rather than webdriver.display
+//        System.setProperty(ChromeDriverService.CHROME_DRIVER_LOG_PROPERTY,
+//                PropertiesHolder.getProperty("webdriver.log"));
+//        driverService = ChromeDriverService.createDefaultService();
+
+        @SuppressWarnings("deprecation")
+        File chromeDriver = getChromeDriver();
         driverService =
                 new ChromeDriverService.Builder()
                         .usingDriverExecutable(
```

Comment:
On 2014-12-22 14:05, Damian Jansen wrote:
>> +        @SuppressWarnings("deprecation")
>> +        File chromeDriver = getChromeDriver();
>>          driverService =
>>                  new ChromeDriverService.Builder()
>>                          .usingDriverExecutable(
> 
> Could put all on one line, like
> 
>     .usingDriverExecutable(getChromeDriver())
> 

I wanted to suppress the deprecation warning just for that line.


-- 
Sean Flanigan

Senior Software Engineer
Engineering - Internationalisation
Red Hat